app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/singleitemcheckout/SingleItemCheckoutDestination.kt

```diff
@@ -89,56 +105,175 @@ private fun SingleItemCheckoutScreen(
+      endSlot = {
+        Row(
+          horizontalArrangement =
+            Arrangement.End,
+          verticalAlignment = Alignment.CenterVertically,
+        ) {
+          val explanationText = when (uiState.compensation) {
+            is ClaimFlowDestination.Compensation.Known.RepairCompensation -> stringResource(
+              id = R.string.CLAIMS_CHECKOUT_REPAIR_CALCULATION_TEXT,
+            )
+            is ClaimFlowDestination.Compensation.Known.ValueCompensation -> stringResource(
+              id = R.string.CLAIMS_CHECKOUT_NO_REPAIR_CALCULATION_TEXT,
+            )
+          }
+          val title = stringResource(R.string.CLAIMS_CHECKOUT_COUNT_TITLE)
+          IconButton(
+            onClick = {
+              bottomSheetText = Pair(title, explanationText)
+            },
+            modifier = Modifier
+              .size(40.dp)
+              .padding(end = 8.dp),
+          ) {
+            Icon(
+              imageVector = Icons.Hedvig.InfoFilled,
+              contentDescription = stringResource(R.string.REFERRALS_INFO_BUTTON_CONTENT_DESCRIPTION),
+              modifier = Modifier.size(16.dp),
+            )
+          }
+        }
+      },
     )
-    Spacer(Modifier.height(8.dp))
     CompositionLocalProvider(LocalContentColor provides MaterialTheme.colorScheme.onSurfaceVariant) {
       Column(sideSpacingModifier) {
-        val pairs = listOf(
-          stringResource(R.string.KEY_GEAR_ITEM_VIEW_VALUATION_PAGE_TITLE) to uiState.price.toString(),
-          stringResource(R.string.claims_payout_age_deduction) to "-" + uiState.depreciation.toString(),
-          stringResource(R.string.claims_payout_age_deductable) to "-" + uiState.deductible.toString(),
-        )
+        val pairs = when (uiState.compensation) {
+          is ClaimFlowDestination.Compensation.Known.RepairCompensation,
+          -> listOf(
+            stringResource(
+              R.string.CLAIMS_CHECKOUT_REPAIR_TITLE,
+              uiState.modelDisplayName,
+            ) to uiState.compensation.repairCost.toString(),
+            stringResource(R.string.claims_payout_age_deductable) to "-" + uiState.compensation.deductible.toString(),
+          )
+
+          is ClaimFlowDestination.Compensation.Known.ValueCompensation -> listOf(
+            stringResource(R.string.KEY_GEAR_ITEM_VIEW_VALUATION_PAGE_TITLE) to uiState.compensation.price.toString(),
+            stringResource(R.string.claims_payout_age_deduction) to "-" + uiState.compensation.depreciation.toString(),
+            stringResource(R.string.claims_payout_age_deductable) to "-" + uiState.compensation.deductible.toString(),
+          )
+        }
         for ((left, right) in pairs) {
-          Row {
-            Text(left, Modifier.weight(1f))
-            Text(right, Modifier.weight(1f), textAlign = TextAlign.End)
-          }
+          HorizontalItemsWithMaximumSpaceTaken(
+            startSlot = {
+              Text(left)
+            },
+            endSlot = {
```

Comment:
We probably want some sane space between here too. I know it's not really evident in the design for all these scenarios, since they rarely show the edge cases, but even a sane 8.dp as space between is more or less always the right choice when having texts side-by-side like that.

Reply:
maybe we put it as default for HorizontalItemsWithMaximumSpaceTaken, no?

Reply:
I've thought of it before too, but we don't always use this just for texts being laid out side-by-side, and in those scenarios it would be confusing to have some padding that you did not specify, where you have to pass `spaceBetween = 0.dp`. It feels odd to pass the 0.dp explicitly in a way.